src/applications/vre/28-8832/components/ReadOnlyUserDescription.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+
+const ReadOnlyUserDescription = props => {
+  const [isLoggedIn, setIsLoggedIn] = useState();
+  useEffect(
+    () => {
+      setIsLoggedIn(props.isLoggedIn);
+    },
+    [props],
+  );
+  const display = {
+    'Your first name': props?.profile?.userFullName?.first || '',
+    'Your middle name': props?.profile?.userFullName?.middle || '',
+    'Your last name': props?.profile?.userFullName?.last || '',
+    Suffix: props?.profile?.userFullName?.suffix || '',
+    'Date of birth': props?.profile?.dob || '',
```

Comment:
With accessibility in mind, the date should be fully spelled out, e.g. `May 4, 1933`